robot/Cumulus/resources/NPSP.robot

```diff
@@ -183,7 +183,28 @@ Run Donations Batch Process
     Wait For Batch To Process    RLLP_OppContactRollup_BATCH        Completed
     Wait For Batch To Process    RLLP_OppHouseholdRollup_BATCH      Completed
     Wait For Batch To Process    RLLP_OppSoftCreditRollup_BATCH     Completed
-    
+
+Validate Batch Process When CRLP Checked
+    Open NPSP Settings          Bulk Data Processes                Rollup Donations Batch
+    Click Settings Button       idPanelOppBatch                    Run Batch
+
+    Wait For Batch To Process    CRLP_Account_SoftCredit_BATCH            Completed
+    Wait For Batch To Process    CRLP_RD_BATCH                            Completed
+    Wait For Batch To Process    CRLP_Account_AccSoftCredit_BATCH         Completed
+    Wait For Batch To Process    CRLP_Contact_SoftCredit_BATCH            Completed
+    Wait For Batch To Process    CRLP_Account_BATCH                       Completed
+    Wait For Batch To Process    CRLP_Contact_BATCH                       Completed
+
+Run Donations Batch Process
+    Go To Page                   Custom                         NPSP_Settings
+    Open Main Menu               Donations
+    Click Link With Text         Customizable Rollups
+    ${crlp_enabled} =            Check Crlp Not Enabled By Default
+
+    #Open NPSP Settings and run Rollups Donations Batch job Validate the batch jobs completeness based accordingly
+    Run Keyword if      ${crlp_enabled} != True
+        ...             Validate Batch Process When CRLP Unchecked
+        ...     ELSE    Validate Batch Process When CRLP Checked
```

Comment:
I think this be wrapped into a keyword and pass the boolean value. I see scope for reusability of this keyword to enable settings of similar kind.

Reply:
Understand. But I feel that in this situation it would be overcomplicating it. Run Keyword IF itself explains the logic and is hidden in the NPSPY.Robot file. The actual test just calls this keyword. My concern is if I have to design a generic keyword
I will need a method to take in the parameters (condition), as conditions can vary may not be boolean all the time and the keywords to execute both if the condition satisfies and if the condition does not satisfy.